1.7/administration/securing-your-cluster.md

```diff
@@ -29,6 +29,44 @@ the other nodes in the cluster via URL routing. For security, the DC/OS cloud
 template allows configuring a whitelist so that only specific IP address
 ranges are permitted to access the admin zone.
 
+#### Steps for Securing Admin Router
+
+By default, Admin Router will permit unencrypted HTTP traffic. This is not
+considered secure, and you must provide a valid SSL certificate and redirect
+all HTTP traffic to HTTPS in order to properly secure access to your cluster.
```

Comment:
We're using TLS by default in Mesos/libprocess, although you can explicitly enable SSLv3 if you must.